debian/README

```diff
@@ -0,0 +1,84 @@
+# llmnrd - Link-Local Multicast Resolution Daemon
```

Comment:
Would a symlink work? I want the readme to be packaged and accessible from the filesystem.